src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/LoanPendingRequestAmount.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { List } from 'semantic-ui-react';
+import Overridable from 'react-overridable';
+
+export class LoanPendingRequestAmount extends Component {
+  render() {
+    const { circulation } = this.props;
+
+    if (circulation.pending_loans_count < 1) return null;
```

Comment:
I would recommend to install [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) if not already done and maybe set autoformat on save (see `Format On Save` section in the doc in the link above) cc @Pineirin